src/components/ChatLog.js

```diff
@@ -0,0 +1,39 @@
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+import './ChatLog.css';
+import React from 'react';
+
+const ChatLog = ({ entries, calculateLikedCount,colorLocal,colorRemote }) => {
```

Comment:
```suggestion
const ChatLog = ({ entries, calculateLikedCount, colorLocal, colorRemote }) => {
```
make sure you're keeping good code style with your spacing 😌